### PR TITLE
python-cli: show test names by default in activity

### PR DESCRIPTION
**Pull request by mireaulf**
Closes https://github.com/preludeorg/libraries/issues/344

### CODE REVIEW

File: python/cli/prelude_cli/views/detect.py [hunk 1]
```diff
@@ -172,8 +174,18 @@ def describe_activity(controller, days, view, tests, tags, endpoints, dos, statu
...
     report = Table()
+    build_tests = dict()
+
+    def get_test(test_id):
+        if show_test_id or not test_id:
+            return test_id
+        if not build_tests:
+            build = BuildController(account=controller.account)
+            build_tests.update({test['id']: test['name'] for test in build.list_tests()})
+        return build_tests[test_id]
```
Comment (makk94) [inline python/cli/prelude_cli/views/detect.py:185]: print DELETED if test_id isn't in list_tests

Comment (makk94) [reply python/cli/prelude_cli/views/detect.py:185]: at least that's what print somewhere else in our API

File: python/cli/prelude_cli/views/detect.py [hunk 2]
```diff
@@ -151,14 +151,16 @@ def add_recommendation(controller, title, description):
 @click.option('--endpoints', help='comma-separated list of endpoint IDs', type=str)
 @click.option('--dos', help='comma-separated list of DOS', type=str)
 @click.option('--statuses', help='comma-separated list of statuses', type=str)
+@click.option('-i', '--show-test-id', help='show test id instead of test names', is_flag=True, type=bool, default=False)
```
Comment (privateducky) [inline python/cli/prelude_cli/views/detect.py:154]: can we make this '-e' for '--expand' ?

File: python/cli/prelude_cli/views/detect.py [hunk 3]
```diff
@@ -172,8 +174,18 @@ def describe_activity(controller, days, view, tests, tags, endpoints, dos, statu
 
     raw = controller.describe_activity(view=view, filters=filters)
     report = Table()
+    build_tests = dict()
+
+    def get_test(test_id):
+        if show_test_id or not test_id:
+            return test_id
+        if not build_tests:
```
Comment (privateducky) [inline python/cli/prelude_cli/views/detect.py:182]: this is a lot of work; let's just do this in 2 lines: 1) get all tests 2) add to dict

Comment (mireaulf) [reply python/cli/prelude_cli/views/detect.py:182]: What dict? Also I don't want to do that extra api call for views that don't need it which is the majority of them

Comment (mireaulf) [reply python/cli/prelude_cli/views/detect.py:182]: I'm trying to not duplicate code here, I would have the same block of code for all views returning test ids

Comment (privateducky) [reply python/cli/prelude_cli/views/detect.py:182]: hmm; my issue here isn't about normalizing the code, it's the fact ``build.list_tests()`` already returns a dict of all tests, so there is no need to move a test from that collection to a new ``build_tests`` dict on demand. just look the value up in the dictionary.

**Comment by privateducky** [comment]
@mireaulf closing in favor of https://github.com/preludeorg/libraries/pull/366/files

note the difference. just a simple conversion lambda does the trick.

**Pull request closed without merging**